ubuntu-advantage

```diff
@@ -94,102 +94,76 @@ Commands:
  enable-fips <token>       enable the FIPS PPA repository and install,
                            configure and enable FIPS certified modules
  disabe-fips               currently not supported
+
 EOF
+    exit 1
 }
 
+main() {
+    local command="$1"
 
-case "$1" in
-    enable-livepatch)
-        check_user
-        livepatch_check_support
-        token="$2"
-        if ! livepatch_validate_token "$token"; then
-            error_msg "Invalid or missing Livepatch token"
-            error_msg "Please visit https://ubuntu.com/livepatch to obtain a Livepatch token."
-            exit 3
-        fi
-        livepatch_enable "$token"
-        ;;
-
-    disable-livepatch)
-        check_user
-        livepatch_check_support
-        remove_snap="no"
-        if [ -n "$2" ]; then
-            if [ "$2" = "-r" ]; then
-                remove_snap="yes"
+    local service
+    service=$(service_from_command "$command")
+    # if the command contains a service name, check that it's valid
+    if [ "$service" ] && ! name_in_list "$service" "$SERVICES"; then
+        usage
```

Comment:
before displaying the usage(), we should print why we are doing it. i.e., "echo invalid comand $command"

Reply:
done